### PR TITLE
Bump up the version for ROCm-5.1.3 release

### PR DESCRIPTION
**Pull request by srekolam**
updates to ROCm recipes for 5.1.3 release.

### CODE REVIEW

File: var/spack/repos/builtin/packages/rocm-openmp-extras/package.py [hunk 1]
```diff
@@ -41,7 +42,8 @@
     "50e9e87ecd6b561cad0d471295d29f7220e195528e567fcabe2ec73838979f61",
     "83ed7aa1c9322b4fc1f57c48a63fc7718eb4195ee6fde433009b4bc78cb363f0",
     "49cfa8f8fc276ba27feef40546788a2aabe259a924a97af8bef24e295d19aa5e",
-    "47dbcb41fb4739219cadc9f2b5f21358ed2f9895ce786d2f7a1b2c4fd044d30f"
+    "47dbcb41fb4739219cadc9f2b5f21358ed2f9895ce786d2f7a1b2c4fd044d30f",
+    "c41958560ec29c8bf91332b9f668793463904a2081c330c0d828bf2f91d4f04e",
```
Comment (estewart08) [inline var/spack/repos/builtin/packages/rocm-openmp-extras/package.py]: Not a deal breaker here, but there is an extra comma on the last hash.

Comment (srekolam) [reply var/spack/repos/builtin/packages/rocm-openmp-extras/package.py]: thx for noticing it ..correcting it

**Review by tldahlgren — DISMISSED** [review]
Confirmed (major) sha256.

**Comment by eugeneswalker** [comment]
@spackbot re-run pipeline

**Comment by eugeneswalker** [comment]
@scottwittenburg I can't view the failed pipeline here, nor can I restart it. Do you know why?

**Comment by tldahlgren** [comment]
@spackbot run pipeline

**Comment by scottwittenburg** [comment]
> I can't view the failed pipeline here, nor can I restart it. Do you know why?

Based on the time of your comment, it looks like you might have tried when we had gitlab CI/CD turned off for the switch over to signed public binaries.  We're still finishing that up and have not yet turned the sync script back on which pushes branches from github to gitlab and posts status on PRs.  But you should be able to look at the failed pipeline at least now.

**Comment by srekolam** [comment]
@tldahlgren  , resolved one merge conflict with the latest develop branch. Can you please review this again.

**Comment by srekolam** [comment]
@tldahlgren can i get some review and approvals for this PR ..Thanks.

**Review by tldahlgren — APPROVED** [review]

**Pull request merged**